content/news/2021-05-05-calling-for-translation-volunteers/index.id.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Panggilan untuk Relawan Penerjemah Video"
+date: 2021-05-05T12:01:51+09:00
+authors:
+    - name: Youngbin Han
+      bio: Penyelenggara, Ubucon Asia 2021/Penasehat, Komunitas Ubuntu Korea
+      email: youngbin@ubuntu-kr.org
+      launchpad: sukso96100
+      github: sukso96100
+      profile: https://avatars2.githubusercontent.com/u/1916739?s=460&v=4
+      linkurl: https://youngbin.xyz
+      linklabel: Website
+---
+
+Saat kami mulai [Panggilan untuk Pembicara](../ 2021-05-05-call-for-speaker), kami sekarang juga memanggil relawan penerjemahan mulai hari ini.
```

Comment:
```suggestion
Saat kami mulai [Panggilan untuk Pembicara](../2021-05-05-call-for-speakers), kami sekarang juga memanggil relawan penerjemahan mulai hari ini.
```
This is a relative path to url of other post, there should be no space unless the path has spaces.